packages/pci-sdk-iframe/src/apiClient/errors.service.ts

```diff
@@ -0,0 +1,26 @@
+interface I401Error {
+	message: string;
+	code: number;
+}
+
+async function get401ResponseMessage(res: Response): Promise<string> {
```

Comment:
🔴 We are leaking an implementation detail into the message parser.

IMO the message parser should receive a number a return a string / Error

Reply:
> I think we could refactor the apiClient code a little bit using async instead promises

@IagoLast I think this is the solution. Eslint was not being my friend during this process, which is why I ended up leaking the whole response into the parser.